core-js/src/main/javascript/cdf/components/ToggleButtonBaseComponent.js

```diff
@@ -90,10 +90,10 @@ define(['../lib/jquery', './InputBaseComponent'], function($, InputBaseComponent
         input.appendTo(li);
         li.append($("<label/>")
           .attr({'for': myself.name + i})
-          .text(myArray[i][1]));
+          .html(myArray[i][1]));
         selectHTML
           .append(li)
-          .append((myself.separator == undefined || myself.separator == null || myself.separator == "null") 
```

Comment:
Can we please dont have this change? just like we talked on the meeting this week